code/modules/events/spacevine.dm

```diff
@@ -371,14 +371,16 @@
 	eat(user)
 
 /datum/spacevine_controller
+	var/datum/round_event/spacevine/linkedEvent
```

Comment:
This seems uneccesary, you have the reference to the controller, just retrieve first vine after spawning.